content/graph/CompressTree.h

```diff
@@ -2,12 +2,13 @@
  * Author: Simon Lindholm
  * Date: 2016-01-14
  * License: CC0
- * Description: Given a rooted tree and a subset S of nodes, compute the minimal
- * subtree that contains all the nodes by adding all (at most $|S|-1$)
- * pairwise LCA's and compressing edges.
+ * Description: Given a tree $T$ rooted at 0, and a subset of nodes $X$ returns
+ * tree with vertex set $\{\text{lca}(x,y):x\in X,y\in X\}$ and edges between
```

Comment:
Agregar espacios en la fórmula latex: $\{\text{lca}(x,y) : x \in X, y \in X\}$
No cambia nada, pero para que sea mas legible en el código